src/Plugin/EndpointCheck/Ping.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * contains \Drupal\relaxed\Plugin\Endpoint\WorkspaceEndpoint
+ */
+
+namespace Drupal\relaxed\Plugin\EndpointCheck;
+
+use Drupal\relaxed\Entity\EndpointInterface;
+use Drupal\relaxed\Plugin\EndpointCheckBase;
+
+/**
+ * @EndpointCheck(
+ *   id = "ping",
+ *   label = "Ping endpoint"
+ * )
+ */
+Class Ping extends EndpointCheckBase {
+
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->execute($configuration['endpoint']);
```

Comment:
I do have some reasoning behind this, which I could explain, but as the code developed I don't think the argument is valid any more.

Commit coming.